src/System.Net.Primitives/tests/FunctionalTests/System.Net.Primitives.Functional.Tests.csproj

```diff
@@ -42,6 +43,10 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
+    <!-- TODO: Remove this reference once packages are updated -->
```

Comment:
> Nope

Line 50 seems to point to a locally built pkgproj which, I expect contains the latest APIs.

> This is telling the test to use the local pkgproj, so it'll end up compiling against the ref built from that and running with the src built from that.

Backing up a bit as I think I misunderstood the reason this is here: Why is this reference to Requests needed for a Primitives contract test?